src/sentry/api/endpoints/auth_index.py

```diff
@@ -245,13 +245,11 @@ def put(self, request: Request) -> Response:
                     if not Authenticator.objects.filter(
                         user_id=request.user.id, type=U2fInterface.type
                     ).exists():
-                        return Response(
-                            {"detail": {"code": "no_u2f"}}, status=status.HTTP_403_FORBIDDEN
-                        )
+                        return Response({"detail": "no_u2f"}, status=status.HTTP_403_FORBIDDEN)
```

Comment:
`detail` is usually a descriptive sentence. i think the reason they use `code` is for shorthand e.g. `no_u2f`

Reply:
ah ok didn't know that, will switch it back

Reply:
This is also a contract change, will this be handled safely?

Reply:
Changed back